src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -172,19 +172,27 @@ private WebDriver createWebDriver(TestCleaner cleaner, TestName testName) throws
             if (StringUtils.isBlank(u)) {
                 throw new Error("remote-webdriver-firefox requires REMOTE_WEBDRIVER_URL to be set");
             }
-            return new RemoteWebDriver(
-                    new URL(u), //http://192.168.99.100:4444/wd/hub
-                    buildFirefoxOptions(testName)
-            );
+            {
+                RemoteWebDriver driver =  new RemoteWebDriver(
+                        new URL(u), //http://192.168.99.100:4444/wd/hub
+                        buildFirefoxOptions(testName)
+                );
+                driver.setFileDetector(new LocalFileDetector());
+                return driver;
```

Comment:
nit: Extract to a method like `private static RemoteWebDriver buildRemoteWebDriver(String url, Capabilities capabilities)`?

Reply:
addressed in 61b364e